pkg/webhook/resources/volumeimportsource/validator.go

```diff
@@ -0,0 +1,69 @@
+package volumeimportsource
+
+import (
+	"fmt"
+	"reflect"
+
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
+	werror "github.com/harvester/harvester/pkg/webhook/error"
+	"github.com/harvester/harvester/pkg/webhook/types"
+)
+
+const (
+	BlankSourceName = "filesystem-blank-source"
```

Comment:
Use `util.ImportSrourceBlank`?

Reply:
I remove this file because the volumeImportSource now created by controller.